prevent-broker-scale-down-if-contains-partition .md

```diff
@@ -0,0 +1,49 @@
+
+# Prevent scale down of the brokers if they contain replicas
+
+This proposal is about adding the mechanism for preventing the scale down of the brokers in the cluster if one or more replicas are hosted on it.
+In the future, when we introduce the automatic re-balancing when scaling up/down the Kafka Cluster ([proposal](https://github.com/strimzi/proposals/blob/main/035-rebalance-types-scaling-brokers.md)), this mechanism would help to prevent any data loss when we are scaling down the brokers when they have some partitions assigned to them.
```

Comment:
It think this is a good feature to have weather we have auto-rebalancing or not. Warning a user of dataloss would be a good for robustness.